org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/CodeLensHandler.java

```diff
@@ -78,11 +78,25 @@ public CodeLens resolve(CodeLens lens, IProgressMonitor monitor) {
 			String command = null;
 			List<Location> locations = null;
```

Comment:
I'd rather check if != null before returning (around L99), in case someone returns null locations in the future.